app/javascript/components/project-submissions/components/submissions-list.jsx

```diff
@@ -67,7 +67,13 @@ const SubmissionsList = ({
               most liked submissions -
               {' '}
             </span>
-            <a data-test-id="view-all-projects-link" href={allSubmissionsPath}> View full list of solutions</a>
+            <a
+              className="text-gold"
```

Comment:
I'm assuming this is one of the apply directives we want to keep - let me know if we want to make it a Tailwind class